cmd/tracee-ebpf/main.go

```diff
@@ -247,6 +252,9 @@ func main() {
 					select {
 					case event := <-cfg.ChanEvents:
 						printer.Print(event)
+						if err := webhook.Send(event); err != nil {
```

Comment:
Adding this here means that for every event we will create and send an http packet, is that right?
If so, it has very bad implications on performance which we should avoid.
Maybe as a first step we should simply make the webhook another printer? WDYT?